code/modules/admin/admin_verbs.dm

```diff
@@ -688,6 +688,23 @@ var/list/admin_verbs_mod = list(
 	log_admin("[key_name(usr)] gave [key_name(T)] the spell [S].")
 	message_admins("<span class='notice'>[key_name_admin(usr)] gave [key_name(T)] the spell [S].</span>", 1)
 
+
+/client/proc/make_invisible(mob/T as mob in mob_list) // -- Urist
+	set category = "Fun"
+	set name = "Make invisible"
+	set desc = "Make a mob completely invisible."
+
+	T.alphas.len++
+	T.alphas[T.alphas.len] = 1
```

Comment:
It would be better for this to have a special key e.g. ``T.alphas["full_admin_invis"]`` or something.

Reply:
Alphas are read directly, not associatively, IU think

Reply:
And if you call the proc again it will make the array yet larger and add a new value. It would be better if It modifies only one value with a specific key instead of creating new ones each time. Am I missing something here?